docs/RFCS/README.md

```diff
@@ -1,209 +1,94 @@
-   - organize a meeting where open concerns are discussed orally,
-   - postponing/cancelling the work altogether.
-
-   In either case, the process to push for conclusion should seek
-   approval to conclude from the reviewer(s) who have opened the
-   discussions so far, allowing sufficient time for them to evaluate
-   the proposed conclusion.
-
-   Often disagreement on an RFC can be resolved by holding a meeting
-   between the interested parties to achieve consensus. The meeting notes
-   or a summary should be added to the RFC comments.
-
-7. At the point where the author(s) and/or shepherd
-   estimate there is consensus to proceed with the project, begin the
-   final comment period (FCP) by posting a comment on the PR.
-   For RFCs with lengthy discussion, the motion to FCP is usually preceded
-   by a summary comment laying out the current state of the
-   discussion and major tradeoffs/points of disagreement. The FCP
-   should last a minimum of two working days to allow collaborators
-   to voice any last-minute concerns.
-
-   In most cases, the FCP period is quiet, and the RFC is merged.
-   However, sometimes substantial new arguments or ideas are raised,
-   the FCP is canceled, and the RFC goes back into development mode.
-
-8. If there is still consensus to proceed after the FCP:
+# About This Directory
+
+This directory contains RFCs (design documents) that describe proposed
+major changes to CockroachDB.
+
+# The Why of RFCs
+
+An RFC provides a high-level description of a major change or
+enhancement to Cockroach. The high-level description allows a reviewer
+to critique and poke holes in a design without getting lost in the
+particulars of code.
+
+An RFC is a form of communication aimed at both spreading and
+gathering knowledge, though it is not the sole means of accomplishing
+either task. Prototypes, tech notes, github issues, comments in code,
+commit messages and in-person discussions are valid alternatives
+depending on the situation.
+
+At its best, an RFC clearly and concisely describes the high-level
+architecture of a project giving confidence to all involved. At its
+worst, an RFC focuses on unimportant details, fosters discussion that
+stymies progress, or demoralizes the author with the complexity of
+their undertaking.
+
+# The When and How of RFCs
+
+When to write an RFC is nuanced and there are no firm rules. General
+guidance is to write an RFC before embarking on a significant or
+complex project that will be spread over multiple pull requests (PRs),
+and when multiple alternatives need to be considered and there is no
+obvious best approach. A project involving multiple people is a good
+signal an RFC is warranted. (Similarly, a project worthy of an RFC
+often requires multiple engineers worth of effort). Note that this
+guidance is intentionally vague. Many complex projects spread over
+multiple PRs do not require an RFC, though they do require adequate
+communication and discussion.
```

Comment:
When there are a number of alternatives where implementing each alternative could take a lot of time it is worth discussing it in an RFC